tools/orca/src/generate.rs

```diff
@@ -122,7 +123,7 @@ pub async fn populate(_client: &KanidmOrcaClient, profile: Profile) -> Result<St
 
         let roles = BTreeSet::new();
 
-        let model = Model::Basic;
+        let model = Model::Writer;
```

Comment:
This needs to be configurable ....

Reply:
Yeah I think that was about time. Should I also add an option to the wizard to choose the model when generating the profile.toml file?

Reply:
Well I did it anyway because I was too amazed by the way enums and clap work together :p